en/docs/install-and-setup/setup/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -99,9 +105,10 @@ Instead of the default `api_params.yaml`, you can a provide custom parameter fil
 
 !!! info
     -   Production/Sandbox backends for each environment can be specified in the parameter file with additional configurations, such as timeouts.
-    -   Under the security field, if the `enabled` attribute is `true`, you must specify the `username`, `password` and the `type` (can be either only `basic` or `digest`). If the `enabled` attribute is `false`, then non of the security parameters will be set. If the `enabled` attribute is not set (blank), then the security parameters in api.yaml file will be considered.
+    -   Under the `security` field, if the `enabled` attribute is `true`, you must specify the `username`, `password` and the `type` (can be either only `basic` or `digest`). If the `enabled` attribute is `false`, then non of the security parameters will be set. If the `enabled` attribute is not set (blank), then the security parameters in api.yaml file will be considered.
```

Comment:
Spelling mistake
Change "non" to "none"

Change [1] to [2] and always follow that concept when writing about files.
- Format : in the \<file-name> file
- Always add the file name as code so that the file name stands out properly.

[1] then the security parameters in api.yaml file will be considered.
[2] then the security parameters in the `api.yaml` file will be considered.

Reply:
Sure. Thanks for pointing that out.
Fixed via https://github.com/wso2/docs-apim/pull/2904/commits/c049f0216cc3f99dbd8b9cfd5d3021c318f365d2